### PR TITLE
[FIX][mass_mailing_custom_unsubscribe] Reasons noupdate

### PR DESCRIPTION
**Pull request by yajo**
After this fix, when you update the addon, you will not lose your customized reasons.

Had to move from csv to XML because there's no noupdate=1 in csv.

@Tecnativa

### CODE REVIEW

File: mass_mailing_custom_unsubscribe/__openerp__.py [hunk 1]
```diff
@@ -23,15 +23,15 @@
 {
     'name': "Customizable unsubscription process on mass mailing emails",
     'category': 'Marketing',
-    'version': '8.0.2.0.0',
+    'version': '8.0.2.0.1',
```
Comment (pedrobaeza) [inline mass_mailing_custom_unsubscribe/__openerp__.py]: You need to update, so 8.0.2.1.0

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Translation doesn't change?

This is needed also for 9.0

**Comment by yajo** [comment]
No translation changed.

**Comment by yajo** [comment]
~BTW this addon needs a full forward migration for v9, not just this patch~

v9 migration is in https://github.com/OCA/social/pull/129, I'll add the patch there

**Comment by pedrobaeza** [comment]
Yeah, it's on https://github.com/OCA/social/pull/129, and it's yours! You don't even remember your own PRs :stuck_out_tongue:

**Comment by rafaelbn** [comment]
@Yajo travis fails

**Comment by yajo** [comment]
I've got 142 open PRs, indeed I don't remember all of them 🤣 

It turns out the branch is broken, I'll try to fix it.

**Comment by yajo** [comment]
Nice, it's Odoo that is broken. See https://github.com/odoo/odoo/pull/15199. Now I have 143 😆.

In short, you can ignore Travis error because it's not related.

**Comment by yajo** [comment]
Now that it's fixed, I squashed and rebased, and bots went 💚 😊

**Review by pedrobaeza — APPROVED** [review]

**Comment by yajo** [comment]
Interesting, if you remove the records and update the addon, they get back in. It seems like noupdate != noreinstall. I'll investigate a little bit more.

**Comment by pedrobaeza** [comment]
I thought you have already put `forcecreate="0"`. That's the way for avoiding recreate them on update.

**Comment by yajo** [comment]
Yay thanks! TIL that. Squashed. Ready to merge then.

**Review by rafaelbn — APPROVED** [review]
Tested

**Pull request merged**